domain/service/service.go

```diff
@@ -441,6 +446,20 @@ func (s *Service) EnablePort(application string, portAddr string, enable bool) e
 	return nil
 }
 
+// Make best effort to make a port address valid
+func ScrubPortString(port string) string{
+	// remove possible protocol at string beginning
+	re := regexp.MustCompile("^(.+://)")
+	scrubbed := re.ReplaceAllString(port, "")
+
+	matched, _ := regexp.MatchString("^[0-9]*$", port)
+	if  matched {
+		scrubbed = fmt.Sprintf(":%s", port)
```

Comment:
This seems fairly dangerous. Why not, say, fmt.Sprintf(":%s", strings.TrimPrefix(port, ":"))?